webanno/webanno
# #1759 User documents view disappear on curation page

User segments were not updated when reloading the SuggestionViewPanel after re-merge or changing preferences. Most likely, because there was an update request when the user segments no longer existed. After  researching the update process of the CurationPage and its components, I've come to the conclusion that it would need some extensive re-factoring (of components, their models and when they are updated) to fix this via the update processes.
So, instead we are using try-catch around the js calls.

**What's in the PR**
* Delay update request
* Added try-catch block to js 


**How to test manually**
* see issue
* Test overall functionality to make sure the app is still working as desired.

**Automatic testing**

**Documentation**
